eng/pipelines/typespec-validation.yml

```diff
@@ -1,32 +1,15 @@
 trigger: none
 
-pr:
-  branches:
-    include:
-    - '*'
-
 jobs:
 - job: Validate_Impacted_Specs
   pool:
     name: azsdk-pool-mms-ubuntu-2204-general
     vmImage: ubuntu-22.04
 
   steps:
-  - script: npm --version --loglevel info
-    displayName: npm --version
-
-  - script: npm ci
-    displayName: npm ci
-
-  - script: npm ls -a
-    displayName: npm ls -a
-    condition: succeededOrFailed()
+  - template: /eng/pipelines/templates/steps/npm-install.yml
 
   - pwsh: |
-      $(Build.SourcesDirectory)/eng/scripts/Validate-TypeSpec.ps1 `
-        $(Build.SourcesDirectory) `
-        "origin/${env:SYSTEM_PULLREQUEST_TARGETBRANCH}" `
-        "${env:SYSTEM_PULLREQUEST_SOURCECOMMITID}" `
-        -GitClean
+      $(Build.SourcesDirectory)/eng/scripts/Validate-TypeSpec.ps1 -GitClean -Verbose
```

Comment:
Why did you add `-Verbose` here?

Reply:
I had the diff helpers output more details via Write-Verbose and at least for now I wanted that output.